src/components/wallet/methods/updateAccount.ts

```diff
@@ -7,19 +7,32 @@ import {
 import { handleError } from '@services/error'
 import { Wallet } from '../wallet'
 
+async function claimableBalanceQuery(wallet: Wallet) {
+  try {
+    let resp = await wallet.server
+      .claimableBalances()
+      .claimant(wallet.account.publicKey)
+      .call()
+    resp.records.forEach((record) => {
+      wallet.logger.response(
+        'Claimable Balances Available ',
+        loOmit(record, ['_links', 'paging_token', 'self'])
+      )
+    })
+  } catch (err) {
```

Comment:
I think we can make this function simpler by removing the try/catch and letting `updateAccount` call `handleError()`

Reply:
Good call